tools/ci/generate_sql_scripts.py

```diff
@@ -39,24 +44,36 @@
 scriptLines = [
     "#!/bin/bash\n",
     "set -euo pipefail\n"
+    "python3 -m pip install setuptools\n" # Yes I know you can PIP multiple things but they need to happen in this order
+    "python3 -m pip install mysql-connector\n"
     "mysql -u root -proot < tools/ci/sql_v0.sql\n"
 ]
 
 # And write the files and tell them to be used
 for file in orderedSqlFiles:
-    inFile = open("SQL/updates/" + file, "r")
-    fileLines = inFile.readlines()
-    inFile.close()
-    # Add in a line which tells it to use the feedback DB
-    fileLines.insert(0, "USE `feedback`;\n")
+    if file.endswith(".py"):
+        # Begin snowflakery
+        if file == "16-17.py":
+            scriptLines.append("python3 SQL/updates/" + str(file) + " 127.0.0.1 root root feedback feedback round\n")
```

Comment:
```suggestion
            scriptLines.append("python3 SQL/updates/%s 127.0.0.1 root root feedback feedback round\n" % file)
```
or if the %s is too smol and sneeki for you:
```suggestion
            scriptLines.append("python3 SQL/updates/{file} 127.0.0.1 root root feedback feedback round\n".format(file=file))
```
Or, again, with the fancy f-syntax:
```suggestion
            scriptLines.append("python3 SQL/updates/{file} 127.0.0.1 root root feedback feedback round\n")
```

Reply:
Ditto